scripts/components/LviHomeProducts.ts

```diff
@@ -43,6 +43,7 @@ export default class LviHomeProducts extends LviHomeProductsDesign {
     private initGridView() {
         this.gvProducts.layoutManager.onItemLength = () => HALF_OF_SCREEN_WIDTH;
         this.gvProducts.onItemBind = (GridViewItem: GviProductItem, productIndex: number) => {
```

Comment:
We should use camelCase for local variables. @suleymanbasbug

```ts
this.gvProducts.onItemBind = (gridViewItem: GviProductItem, productIndex: number) => {
```

Reply:
OK.